mobile/lib/utils/file_download_util.dart

```diff
@@ -267,10 +267,7 @@ Future<void> _saveLivePhotoOnDroid(
   savedAsset = (await (PhotoManager.editor.saveVideo(
     video,
     title: videoTitle,
-  )));
-  if (savedAsset == null) {
-    throw Exception("Failed to save video of live photo");
-  }
+  )).catchError((_) => throw Exception("Failed to save video of live photo")));
```

Comment:
Is there a reason to add these changes?
Instead of catchError, i would probably prefer try catch for better readability.

Reply:
It is due to the upgrade, basically now savedAsset will never be null, so I had to catch and throw exceptions myself.